nixos/modules/services/hardware/nvidia-container-toolkit/cdi-generate.nix

```diff
@@ -0,0 +1,37 @@
+{ config, pkgs }: let
+  mountOptions = { options = ["ro" "nosuid" "nodev" "bind"]; };
+  mounts = [
+    { hostPath = "/run/current-system/sw/bin/nvidia-cuda-mps-control";
+      containerPath = "/usr/bin/nvidia-cuda-mps-control"; }
+    { hostPath = "/run/current-system/sw/bin/nvidia-cuda-mps-server";
+      containerPath = "/usr/bin/nvidia-cuda-mps-server"; }
+    { hostPath = "/run/current-system/sw/bin/nvidia-debugdump";
+      containerPath = "/usr/bin/nvidia-debugdump"; }
+    { hostPath = "/run/current-system/sw/bin/nvidia-powerd";
+      containerPath = "/usr/bin/nvidia-powerd"; }
+    { hostPath = "/run/current-system/sw/bin/nvidia-smi";
+      containerPath = "/usr/bin/nvidia-smi"; }
+    { hostPath = "${pkgs.nvidia-container-toolkit}/bin/nvidia-ctk";
+      containerPath = "/usr/bin/nvidia-ctk"; }
+    { hostPath = "${pkgs.glibc}/lib";
+      containerPath = "${pkgs.glibc}/lib"; }
```

Comment:
(not a blocker/not actionable) Not sure if indirection (`/run`) is useful here, it's the (userspace) drivers that we use indirection for, mostly. And for some of those, it's the `booted-system` rather than `current-system` that we'd need: https://github.com/NixOS/nixpkgs/issues/255070.